### PR TITLE
Change all calls to Time.now into Time.current

### PR DESCRIPTION
**Pull request by robbkidd**
Per a few resources' recommendations[1], calls for current time in a Rails
app should be to `Time.current` not `Time.now`.

[1] Resources:
* http://rails-bestpractices.com/posts/2014/10/22/use-time-zone-now-instead-of-time-now/
* http://danilenko.org/2012/7/6/rails_timezones/
* http://winstonyw.com/2014/03/03/time_now_vs_time_zone_now/

Addresses issue #1139

### CODE REVIEW

**Comment by robbkidd** [comment]
In general, I believe the use of `Time.now` was mitigated by [the application config default time zone being set to UTC](https://github.com/chef/supermarket/blob/master/config/application.rb#L68). The setting of values to `Time.current` should not affect things going forward. I'll comment on when a `Time.current` is being used for comparison.

File: lib/supermarket/health.rb [hunk 1]
```diff
@@ -56,7 +56,7 @@ def expired_ocid_tokens
       postgres_health_metric do
         @supermarket[:expired_ocid_tokens] = Account.
           for('chef_oauth2').
-          where('oauth_expires < ?', Time.now).
+          where('oauth_expires < ?', Time.current).
```
Comment (robbkidd) [inline lib/supermarket/health.rb]: Values coming out of the database will be in UTC. ActiveRecord auto-converts time to UTC (per the app config) so there should be no behavioral difference in use of `Time.now` vs `Time.current` in this query.

**Comment by nellshamrell** [comment]
:+1:

**Comment by nellshamrell** [comment]
TY for this!

**Pull request merged**